frontend/apps/home/src/app/tooltip/overlay-style.component.ts

```diff
@@ -0,0 +1,11 @@
+import {Component, ViewEncapsulation} from "@angular/core";
+
+@Component({
+  selector: 'hmd-overlay-styles',
+  template: '',
+  styleUrls: ["overlay-style.component.scss"],
+  encapsulation: ViewEncapsulation.None
```

Comment:
komponent służy tylko do zaimportowania styli z overlay, nie chciałem ich dodawać globalnie w aplikacji tylko mieć dostępne razem z modułem tooltip tam gdzie jest to potrzebne